nengo/tests/test_neurons.py

```diff
@@ -528,3 +531,36 @@ def test_gain_bias(rng, nl_nodirect, generic):
 
     assert np.allclose(max_rates, max_rates0, atol=tolerance)
     assert np.allclose(intercepts, intercepts0, atol=tolerance)
+
+
+def _benchmark(neuron_type, n, d=1, sim_time=0.5, n_trials=10):
+    """Estimates number of milliseconds per simulation time-step."""
+
+    with nengo.Network() as model:
+        nengo.Ensemble(n_neurons=n, dimensions=d, neuron_type=neuron_type)
+
+    with nengo.Simulator(model, progress_bar=False) as sim:
+        sim.step()  # invokes the JIT compilation (cached from here on)
+
+        def to_time():  # timed portion of code
+            sim.run(sim_time, progress_bar=False)
+
+        n_steps = float(sim_time) / sim.dt
+        return 1e3 * timeit.timeit(to_time, number=n_trials) / float(n_trials) / n_steps
+
+
+@pytest.mark.slow
+def test_numbalif_benchmark(plt):
```

Comment:
Should be marked with @pytest.mark.noassertions
```suggestion
@pytest.mark.noassertions
def test_numbalif_benchmark(plt):
```